package.json

```diff
@@ -107,9 +107,6 @@
   "engines": {
     "node": "^6.x.x || ^7.x.x"
   },
-  "jest": {
-    "notify": true
```

Comment:
Is this removed because we set those settings somewhere else? I like the notifications